project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module

```diff
@@ -272,9 +272,12 @@ function c4m_restful_quick_post_form($group_id) {
     }
   }
 
+  $user_may_create_discussion = c4m_user_is_site_admin() ?
```

Comment:
This variable indicates if user has a permission to create discussion.
Comment that comes next explains how it's used with relation to QP form:

```
  // If all resources are turned off, or user is not allowed to create a
  // discussion, disable the QP form.
```